### ISSUE TITLE
Is this Version Compatible with Multiple Size of Devices?

### ISSUE THREAD

**Issue by gOzaru** [PanaraStudios/panaradialogs#1]
Hello, I want to ask if this version is compatible and adjustable for all kinds of size in devices?
If it is not, is there anyway to customize it to fit our needs?
Thank you very much.

**Comment by CodeWithKamal** [comment]
Hello @gOzaru , Current version doesn't supports size adjustments, We are working on it to make it fully customisable, it will be out soon in few days in next version. stay tuned!

**Comment by CodeWithKamal** [comment]
Hello @gOzaru  , It's auto responsive for web, desktop & mobile but doesn't supports size adjustments, We are working on it to make it fully customisable, it will be out soon in few days in next version. stay tuned!

**Comment by gOzaru** [comment]
I got an error where the image file is no available....
Maybe you can fix this.
I combined this widget with Flutter Screenutil and it managed to get resized without doing anything. 
Well done.

**Comment by CodeWithKamal** [comment]
@gOzaru We have fixed the image error, please update the version to the latest from your pubspec.yaml file.

**Comment by gOzaru** [comment]
Okay, thank you for the hard work. 
All authors of Dart packages which I used inside the project will be regarded as contributors and they will receive rewards from company later based on the net revenue. I will let you know as you are also considered as one of them. 
Keep making a good package.

Oh yeah, 
I've been looking also for tutorials of how to run Flutter with Cloud Run. If you know one, let me know. All tutorials authors which I learned from whether it is on YouTube or website [medium, etc.], they will also receive this same reward.
